AppMetrics/FeedbackDialog.qml

```diff
@@ -180,7 +180,7 @@ Dialog {
 
                             background: Rectangle {
                                 anchors.fill: parent
-                                color: Singletons.Config.buttonStates(control, "major")
+                                color: Singletons.Config.buttonStates(parent, "major")
```

Comment:
Please refer to the parent element with its `id` instead of `parent`. This increases readability.